components.json

```diff
@@ -231,11 +231,7 @@
 		},
 		"django": {
 			"title": "Django/Jinja2",
-			"require": "markup",
-			"peerDependencies": [
-				"css",
-				"javascript"
-			],
```

Comment:
I think we don't want to remove the `peerDependencies` here.

Reply:
Yes we do because they aren't really peer dependencies in the sense that Django modifies JS or CSS. It just has to be loaded after them.

The new implementation doesn't care about the markup language because markup-templating takes care of that, so we can safely remove them.